pkg/rosa/versions.go

```diff
@@ -2,13 +2,25 @@ package rosa
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
+	"github.com/blang/semver"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 )
 
+var MinSupportedVersion = semver.MustParse("4.14.0")
```

Comment:
This seems like something that OCM - not this repo - defines. Why do we have our own minimum?

Reply:
this is mainly for nodePools, OCM does define this but we can't retrieve this value.
`version.HostedControlPlaneEnabled()` will return false for `4.14.0` because only `4.14.5+` are supported for hosted control planes, but nodepools version can actually be `4.14.0`